### PR TITLE
docs(all): update release instructions

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
...
+If there are any failures in the most recent build, address them before
+proceeding with the release.
+
+### Automated Release
+
+If there are changes that have not yet been released a pull request should be
+automatically opened with a title like "chore: release 0.XX.0", where XX is the
+next version to be released. To cut a release approve and merge this pull
+request. Doing so will update the `CHANGES.md`, tag the merged commit with the
+appropriate version , and draft a GitHub release.
```
Comment (tbpg) [inline RELEASING.md:75]: Draft? Does the person doing the release need to do anything?

File: RELEASING.md [hunk 2]
```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
...
+
+Check for failures in the [continuous Kokoro build](http://go/google-cloud-go-continuous).
+If there are any failures in the most recent build, address them before
+proceeding with the release.
+
+### Automated Release
+
+If there are changes that have not yet been released a pull request should be
+automatically opened with a title like "chore: release 0.XX.0", where XX is the
+next version to be released. To cut a release approve and merge this pull
```
Comment (tbpg) [inline RELEASING.md:73]:
```suggestion
next version to be released. To cut a release, approve and merge this pull
```

File: RELEASING.md [hunk 3]
```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
...
-# How to release `cloud.google.com/go`
+## How to release `cloud.google.com/go`
+
+Check for failures in the [continuous Kokoro build](http://go/google-cloud-go-continuous).
+If there are any failures in the most recent build, address them before
+proceeding with the release.
+
+### Automated Release
+
+If there are changes that have not yet been released a pull request should be
```
Comment (tbpg) [inline RELEASING.md:71]: Specify a PR from release-please? Link to those docs?

File: RELEASING.md [hunk 4]
```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
...
+automatically opened with a title like "chore: release 0.XX.0", where XX is the
+next version to be released. To cut a release approve and merge this pull
+request. Doing so will update the `CHANGES.md`, tag the merged commit with the
+appropriate version , and draft a GitHub release.
+
+### Manual Release
```
Comment (tbpg) [inline RELEASING.md:77]: Give some detail about when & why someone would need to do this?

File: RELEASING.md [hunk 5]
```diff
@@ -76,9 +88,11 @@ the failures have been resolved.
    (the `git log` is going to show you things in submodules, which are not going
    to be part of your release).
 1. Edit `CHANGES.md` to include a summary of the changes.
-1. In `internal/version/version.go`, update `const Repo` to today's date with the format `YYYYMMDD`.
+1. In `internal/version/version.go`, update `const Repo` to today's date with
```
Comment (tbpg) [inline RELEASING.md:91]: Do we have a tracking bug for this?

**Review by tbpg — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 1 above)
Comment (codyoss) [reply RELEASING.md:75]: I think the is the terminology used for releases. The button in Github to do this is `Draft a release`. And nope, it is done automagically. Here is an example of an automated github release: https://github.com/googleapis/google-cloud-go/releases/tag/v0.70.0

File: RELEASING.md (hunk 4 above)
Comment (codyoss) [reply RELEASING.md:77]: done

File: RELEASING.md (hunk 5 above)
Comment (codyoss) [reply RELEASING.md:91]: Created: https://github.com/googleapis/google-cloud-go/issues/3109

File: RELEASING.md [hunk 6]
```diff
@@ -53,17 +55,27 @@ of the `cloud.google.com/go` repository root module. Note: releasing
...
+If there are any failures in the most recent build, address them before
+proceeding with the release.
+
+### Automated Release
+
+If there are changes that have not yet been released a pull request should be
+automatically opened with a title like "chore: release 0.XX.0", where XX is the
+next version to be released. To cut a release, approve and merge this pull
+request. Doing so will update the `CHANGES.md`, tag the merged commit with the
+appropriate version , and draft a GitHub release.
```
Comment (tbpg) [inline RELEASING.md:75]:
```suggestion
appropriate version, and draft a GitHub release.
```

File: RELEASING.md (hunk 3 above)
Comment (tbpg) [reply RELEASING.md:71]: Looks like this change wasn't uploaded?

**Review by tbpg — APPROVED** [review]

**Comment by codyoss** [comment]
They are live now, turns out it helps if you `git push` 😄

**Pull request merged**